spec/spec_helper.rb

```diff
@@ -8,9 +8,26 @@
 require 'capybara/rails'
 require 'database_cleaner'
 require 'fabrication'
+require 'selenium/webdriver'
```

Comment:
no, and `capybara/rails` either but where do you want to require them, in a `before(:all)`?